Packages/com.unity.inputsystem/InputSystem/Devices/Commands/InputDeviceCommand.cs

```diff
@@ -28,6 +28,8 @@ namespace UnityEngine.InputSystem.LowLevel
     public struct InputDeviceCommand : IInputDeviceCommandInfo
     {
         internal const int kBaseCommandSize = 8;
+        public const int BaseCommandSize = 8;
```

Comment:
For these kind of changes, please leave comment RE intention here. Something like

```
////TODO: Remove kBaseCommandSize
```